common/export.py

```diff
@@ -381,7 +390,58 @@ def export_json(json_data, filename):
     with open(filename, "w", encoding="utf-8") as file:
         file.write(json_string)
         file.write("\n")  # Add a newline at the end of the file
-    logger.info(f"\nExported {len(json_data)} elements to {filename}")
+
+    logger.info(f"Exported {len(json_data)} elements to {filename}")
+
+
+def format_json(json_path):
+    return subprocess.run(f"npm run fix:prettier {json_path}".split(" "))
```

Comment:
I'm a bit uneasy with a python script calling a npm command, knowing that some npm command may call python scripts as well. I usually prefer things to work one-way

Reply:
I agree but I don't see how to do it otherwise, do you have any idea?  (calling python scripts within npm would be ok anyway)